packages/web-components/fast-components/src/search/index.ts

```diff
@@ -0,0 +1,84 @@
+import { attr } from "@microsoft/fast-element";
+import {
+    Search as FoundationSearch,
+    searchTemplate as template,
+} from "@microsoft/fast-foundation";
+import { Swatch } from "../color/swatch";
+import { fillColor, neutralFillRecipe } from "../design-tokens";
+import { searchStyles as styles } from "./search.styles";
+
+/**
+ * Search appearances
+ * @public
+ */
+export type SearchAppearance = "filled" | "outline";
+
+/**
+ * @internal
+ */
+export class Search extends FoundationSearch {
+    /**
+     * The appearance of the element.
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: appearance
+     */
+    @attr
+    public appearance: SearchAppearance;
```

Comment:
If you set a default for appearance here, you can get rid of the need for connectedCallback() entirely.
```suggestion
    public appearance: SearchAppearance = "outline";
```